src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -273,6 +275,9 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
 
         prepareStudentExamForConduction(request, currentUser, testRun);
 
+        // TODO: filter results in participations and submissions before the end of the test run
```

Comment:
is this TODO still needed?

Reply:
Good catch, no this todo is not needed anymore,  we filter within the `prepareStudentExamForConduction`